src/sonic-frr/Makefile

```diff
@@ -4,20 +4,24 @@ SHELL = /bin/bash
 
 MAIN_TARGET = $(FRR)
 DERIVED_TARGET = $(FRR_PYTHONTOOLS) $(FRR_DBG) $(FRR_SNMP) $(FRR_SNMP_DBG)
-BRANCH = $(shell date +%Y%m%d\.%H%M%S)
+FRR_VERSION = 7.1
```

Comment:
You don't need to define it. It's exported from `rules/frr.mk: export FRR_VERSION FRR_SUBVERSION` along with the subversion.

Reply:
Thanks for your comment! I will remove  it.